product-matrix.json

```diff
@@ -5,18 +5,20 @@
       "version": "3.1",
       "ideaProduct": "android-studio",
       "ideaVersion": "173.4670197",
-      "dartPluginVersion": "173.4700",
+      "dartPluginVersion": "181.4203.498",
       "sinceBuild": "173.0",
-      "untilBuild": "173.*"
+      "untilBuild": "173.*",
+      "filesToSkip": ["src/io/flutter/editor/FlutterCompletionContributor.java"]
     },
     {
       "name": "IntelliJ",
       "version": "3.2",
       "ideaProduct": "android-studio",
-      "ideaVersion": "173.4688006",
-      "dartPluginVersion": "173.4700",
-      "sinceBuild": "173.0",
-      "untilBuild": "173.*"
+      "ideaVersion": "181.4705630",
+      "dartPluginVersion": "181.4203.498",
```

Comment:
In my local copy of Android Studio 3.2, I see `181.2784.29` as the Dart plugin version -

Reply:
Yes, that's true for me as well. But it won't compile with that version.

Reply:
I suspect that we'll need to compile against the version that users will have at runtime. Are these compile issues w/ APIs from the Dart plugin? If so, we can probably resolve that (push changes to those versions).

Reply:
It looks like that version of the Dart plugin does not have the `com.jetbrains.lang.dart.ide.completion.DartCompletionExtension` class.

@pq @alexander-doroshko - the latest AS (3.2 release 10) selects v `181.2784.29` of the Dart plugin, not `181.4203.498`, the version w/ the latest changes.

Reply:
Perhaps we can just exclude `src/io/flutter/editor/FlutterCompletionContributor.java` from this build? We were doing something similar for the 2017.1 builds.

Reply:
Silly me, I knew that. It was added after the version of Dart plugin that works with 3.2 was created.

FWIW the 173 branch of the Dart plugin source has all the right stuff. That's what I use during development, and everything compiles. But there is no plugin distro that matches that source that is selectable for Android Studio 3.2.

There are a number of changes to the analysis server in the Dart plugin that are present in 173 but not 181.2784.29, which means any Flutter plugin we build for 3.2 won't run until the Dart plugin is updated.